x-pack/plugins/security_solution/public/common/utils/timeline/use_show_timeline.tsx

```diff
@@ -6,29 +6,29 @@
  */
 
 import { useState, useEffect } from 'react';
-import { useRouteSpy } from '../route/use_route_spy';
+import { matchPath, useLocation } from 'react-router-dom';
 
-const hideTimelineForRoutes = [`/cases/configure`, '/administration', 'rules/create'];
+const HIDDEN_TIMELINE_ROUTES: readonly string[] = [
+  `/cases/configure`,
+  '/administration',
+  '/rules/create',
+  '/get_started',
```

Comment:
maybe we should get these from`x-pack/plugins/security_solution/common/constants.ts`?

Reply:
we are going to change hook to use `appLinks` paths in the following PR. I'll leave it this way by now.